src/plugins/intel_cpu/src/mkldnn_exec_network.cpp

```diff
@@ -242,6 +258,78 @@ InferenceEngine::Parameter MKLDNNExecNetwork::GetMetric(const std::string &name)
     }
 }
 
+InferenceEngine::Parameter MKLDNNExecNetwork::GetMetric(const std::string &name) const {
+    if (_graphs.empty())
+        IE_THROW() << "No graph was found";
+    // @todo Can't we just use local copy (_cfg) instead?
+    auto graphLock = GetGraph();
+    const auto& graph = graphLock._graph;
+    const auto& config = graph.getProperty();
+
+    if (isLegacyAPI()) {
+        return GetMetricLegacy(name, graph);
+    }
+
+    auto RO_property = [](const std::string& propertyName) {
+        return ov::PropertyName(propertyName, ov::PropertyMutability::RO);
+    };
+
+    if (name == ov::supported_properties) {
+        return std::vector<ov::PropertyName> {
+            RO_property(ov::model_name.name()),
+            RO_property(ov::optimal_number_of_infer_requests.name()),
+            RO_property(ov::streams::num.name()),
+            RO_property(ov::affinity.name()),
+            RO_property(ov::inference_num_threads.name()),
+            RO_property(ov::enable_profiling.name()),
+            RO_property(ov::hint::inference_precision.name()),
+            RO_property(ov::hint::performance_mode.name()),
+            RO_property(ov::hint::num_requests.name()),
```

Comment:
I see some metrics are missed in comparison with Engine::GetMetric. For example range_for_streams, range_for_async_infer_requests and some others. Shouldn't they be included here as well?

Reply:
I am not sure.
I kept legacy behavior here, which is to return only params that has corresponding entity in Config.
As it was discussed we need to have clear view what we should / want to return and what the expectations are.

Reply:
As for those particular parameters I think logically it is correct to return them from engine, so user can check the available ranges before configure them. While there is no reason from user point of view to check those ranges using ExecNetwork since there is no option to reconfigure them.
But again those are just my thoughts
We better to have clear comment/motivation for each particular parameter